src/utils/logging.js

```diff
@@ -22,16 +22,28 @@ function withScope<R>(callback: Scope => R): R {
   return ((ret: $FlowFixMe): R);
 }
 
+type ServerVersionTags = {|
+  rawServerVersion: string | void,
+  coarseServerVersion: string | void,
+  fineServerVersion: string | void,
+|};
+
 /**
  * Get server-version tags at various levels of granularity.
+ *
+ * If the passed server version is falsy, all the tags will be
+ * `undefined`.
  */
-const getServerVersionTags = () => {
-  const zulipVersion = getLoggingContext()?.serverVersion;
-
+export const getServerVersionTags = (zulipVersion: ?ZulipVersion): ServerVersionTags => {
   // Why might we not have the server version? If there's no active
-  // account.
+  // account. N.B: an account may be the active account, but not
+  // logged in.
```

Comment:
This can perhaps point at the glossary entry:
https://github.com/zulip/zulip-mobile/blob/5f454954f/docs/glossary.md#active-account